### PR TITLE
Improve metadata documentation

### PR DESCRIPTION
**Pull request by bkamins**
Fixes https://github.com/JuliaData/DataFrames.jl/issues/3168

### CODE REVIEW

**Comment by bkamins** [comment]
I have added an example how one should check if some key is present in metadata.

**Comment by bkamins** [comment]
I have moved the `in` part as you suggested and re-written it a bit to better match the context.

File: docs/src/lib/metadata.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ Table-level metadata are key-value pairs that are attached to `df`.
 Column-level metadata are key-value pairs that are attached to
 a specific column `col` of `df` data frame.
 
+To check if some key `key` is present in table-level metadata of data frame `df`
+you can write `key in metadatakeys(df)`. Similarly to check if key `key` is present
```
Comment (nalimilan) [inline docs/src/lib/metadata.md:21]:
```suggestion
To check whether some key `key` is present in table-level metadata of data frame `df`
you can write `key in metadatakeys(df)`. Similarly to check whether key `key` is present
```

**Review by nalimilan — APPROVED** [review]

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**